CMakeLists.txt

```diff
@@ -19,6 +19,7 @@ list(APPEND _noahmp_cap_files drivers/nuopc/lnd_comp_kind.F90
 
 # CCPP interface
 list(APPEND _noahmp_ccpp_files drivers/ccpp/noahmpdrv.F90
+	                       noahmp/drivers/ccpp/lnd_iau_mod.F90
```

Comment:
@tsga I think this is wrong. It must be `drivers/ccpp/lnd_iau_mod.F90`. The test still fails.

Reply:
Thank you @uturuncoglu. I just fixed it.